lib/vmdb/settings/database_source.rb

```diff
@@ -3,23 +3,57 @@ class Settings
     class DatabaseSource
       include Vmdb::Logging
 
-      attr_reader :resource
+      attr_reader :resource_instance, :settings_holder_class
```

Comment:
i am trying to have different name for attributes and parameters, it is just looks confusing if parameter shadow attribute